src/components/accessories/appHeader/AppHeader.tsx

```diff
@@ -10,15 +10,24 @@ const AppHeader: FunctionComponent<TProps> = ({
   userCredentials,
   breadcrumbMap,
 }) => {
+  const appHeader = useRef(null);
```

Comment:
Is this `appHeader` ref doing something and I'm missing the point or we can remove it?